apps/www/_blog/2024-01-31-nosql-mongodb-compatibility-with-ferretdb-and-flydotio.mdx

```diff
@@ -0,0 +1,204 @@
+```
+
+FerretDB runs on the default MongoDB port `27017` and also spins up some monitoring tools on port `8080`. Once up and running you can access these at [localhost:8080](http://localhost:8080).
+
+Once up and running, constructing the MongoDB URI is easil:
+
+```bash Terminal
+export MONGODB_URL=mongodb://$DB_USER.$SUPA_PROJECT_REF:$DB_PASSWORD@127.0.0.1:27017/ferretdb?authMechanism=PLAIN
+```
+
+## Test with `mongosh`
+
+If you have MongoDB installed locally on your machine, you can test via `mongosh`, the MongoDB shell.
+
+```bash Terminal
+mongosh '$MONGODB_URL'
+```
+
+If you don't have MongoDB installed locally, you can run the shell via a Docker container:
+
+```bash Terminal
+docker run --rm -it --entrypoint=mongosh mongo \
+ "$MONGODB_URL"
+```
+
+### Insert documents into FerretDB
+
+With `mongosh` running, let's try to insert some documents into our FerretDB instance.
+You are going to insert two footballer data into a `players` collection.
+
+```json5 ferretdb>
+db.players.insertMany([
+   {
+       futbin_id: 3,
+       player_name: "Giggs",
+       player_extended_name: "Ryan Giggs",
+       quality: "Gold - Rare",
+       overall: 92,
+       nationality: "Wales",
+       position: "LM",
+       pace: 90,
+       dribbling: 91,
+       shooting: 80,
+       passing: 90,
+       defending: 44,
+       physicality: 67
+   },
+   {
+       futbin_id: 4,
+       player_name: "Scholes",
+       player_extended_name: "Paul Scholes",
+       quality: "Gold - Rare",
+       overall: 91,
+       nationality: "England",
+       position: "CM",
+       pace: 72,
+       dribbling: 80,
+       shooting: 87,
+       passing: 91,
+       defending: 64,
+       physicality: 82,
+       base_id: 246
+   }
+]);
+```
+
+Great!
+Now when you run `db.players.find()`, it should return all the documents stored in the collection.
+
+### Update document record in FerretDB
+
+Next, you need to update "Giggs" record to reflect his current position as a `CM`.
+To do this, we can just run an `updateOne` command to target just that particular player:
+
+```json5 ferretdb>
+db.players.updateOne(
+    { player_name: "Giggs" },
+    { $set: { position: "CM" } }
+);
+```
+
+Let's query the collection to see if the changes have been made:
+
+```json5 ferretdb>
+db.players.find({player_name: "Giggs"})
+```
+
+You can run many MongoDB operations on FerretDB. See the [list of supported commands](https://docs.ferretdb.io/reference/supported-commands/) in the FerretDB documentation for more.
+
+## Inspect the JSONB data in the Supabase Dashboard
+
+FerretDB stores each collection in a table on the `ferretdb` schema, each document represented by a JSONB entry. You can inspect this in the [Table Editor](https://supabase.com/dashboard/project/_/editor) in your Supabase Dashboard.
+
+![Supabase Dashboard Table Editor Screenshot](/images/blog/getting-started/ferretdb/table-editor.png)
```

Comment:
small improvement could be to open this as a side panel (easier to see the JSON document)